Sources/KlaviyoSwift/StateManagement/StateManagement.swift

```diff
@@ -513,6 +519,16 @@ struct KlaviyoReducer: ReducerProtocol {
                 _ = klaviyoSwiftEnvironment.setBadgeCount(count)
             }
 
+        case .syncBadgeCount:
+            Task {
+                await MainActor.run {
+                    if let userDefaults = UserDefaults(suiteName: Bundle.main.object(forInfoDictionaryKey: "Klaviyo_App_Group") as? String) {
```

Comment:
should this be `klaviyo_badge_autoclearing` like in Sources/KlaviyoCore/KlaviyoEnvironment.swift?

Reply:
casing wise? or are you asking if it should be `klaviyo_badge_autoclearing` and not the app group

Reply:
@kennyklaviyo are you saying this should be in SDK state vs. UserDefaults?

Reply:
ooo sorry didnt realize it was a different string, should this one be lower cased too?

Reply:
yeah it could be lower cased

Reply:
also..does this mean a developer can explicitly set something in `Klaviyo_App_Group` in their Info.plist just like for `klaviyo_badge_autoclearing` and mess with our values?

Reply:
ah gotcha that makes sense then, will this be a breaking change btw? as in if someone just bumped the sdk version on their app, it wouldnt be able to build without these new entries?

Reply:
not breaking, if the app group is not defined in the Info.plist it just doesn't load the User Defaults